### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DaveHOnCode**
Added current date preselected function

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -345,6 +345,17 @@ protected override void OnAfterRender(bool firstRender)
 </DateTimePicker>
 ```
 
+To achieve the same behavior as in the original library (the current date is preselected) the following code can be used:
+
+```html
+<DateTimePicker 
+        DateChangeEvent="(date) => DateChangeEventTest(date)" 
+        From=@DateTime.Now.ToString("yyyy/MM/dd", CultureInfo.InvariantCulture)
```
Comment (yagizhanNY) [inline README.md:353]: It's already defined in [line 20](https://github.com/yagizhanNY/siemens-ix-blazor/blob/896446eced09869a31760560b6db3153a0e26a92/SiemensIXBlazor/Components/DateTimePicker/DateTimePicker.razor.cs#L20). Is it now working as expected?

File: README.md [hunk 2]
```diff
@@ -657,7 +668,7 @@ div[data-tab-content].show {
 ```html
 <Tile Size="TileSize.Medium" Class="mr-1">
     <div slot="header">Tile header</div>
-    <div class="text-l">92.8 �C</div>
+    <div class="text-l">92.8 °C</div>
```
Comment (yagizhanNY) [inline README.md:671]: What did you change in this line? I could not see any changes.

Comment (yagizhanNY) [reply README.md:671]: Oh okay I saw that now.

File: README.md (hunk 1 above)
Comment (DaveHOnCode) [reply README.md:353]: `CultureInfo.InvariantCulture` is missing I think, coz I using the german culture settings

Comment (yagizhanNY) [reply README.md:353]: Hi @DaveHOnCode, will you change the code behind?

Comment (DaveHOnCode) [reply README.md:353]: Yeah I can see about it if I'll get more time

**Pull request closed without merging**